nsot/api/serializers.py

```diff
@@ -468,8 +474,6 @@ def create(self, validated_data):
             validated_data, commit=False
         )
 
-        # Try to populate the related fields and if there are any validation
-        # problems, delete the object and re-raise the error. If not, save the
         # changes.
```

Comment:
I think you forgot to remove this last line of the comment along with the other two lines

Reply:
I actually didn't mean to delete those lines at all.